### DISCUSSION TITLE
Sealed Microsoft.UI.Xaml.Controls.Primitives.RepeatButton

### DISCUSSION THREAD

**Discussion by arivoir** [microsoft/microsoft-ui-xaml#7473] [General]
Unlike Button, RepeatButton is sealed, which forbids inheriting to extend its features. In WPF this same control is not sealed. 

Why? Could the sealed modifier be removed?

**Comment by marcelwgn** [comment]
Unsealing all controls is planned for WinUI 3.X. See #780 for more context.
